.github/workflows/check-sdist.yml

```diff
@@ -19,15 +19,15 @@ jobs:
         # scipy and cython are required to build sdist
         run: |
           python -m pip install --upgrade pip
-          pip install check-manifest scipy cython
+          pip install check-sdist scipy cython
```

Comment:
By reading the [check-sdist](https://github.com/henryiii/check-sdist#configuration) docs, we do not need to install the dependencies:

```suggestion
          pip install check-sdist
```

Reply:
Done